e2e-tests/src/test/java/dev/streamx/cli/StreamxCliPublicationIT.java

```diff
@@ -24,8 +25,10 @@
 public class StreamxCliPublicationIT {
 
   private static final int CLI_SHORT_TIMEOUT_IN_SEC = 2;
-  @ConfigProperty(name = "streamx.cli.e2e.web.delivery.port.url", defaultValue = "http://localhost:8081/")
+  @ConfigProperty(name = "streamx.cli.e2e.web.delivery.port.url", defaultValue = "http://localhost:8087/")
   String webDeliveryPortUrl;
+  @ConfigProperty(name = "streamx.cli.e2e.web.delivery.composite.port.url", defaultValue = "http://localhost:8089/")
```

Comment:
basically it's nginx port, not web.delivery.composite.

I think we can use nginx name as it's our test environment part